shared/node.go

```diff
@@ -183,14 +183,17 @@ func (node *Node) handleEvent(message *messaging.PocketEnvelope) error {
 	case messaging.DebugMessageEventType:
 		return node.handleDebugMessage(message)
 	case messaging.ConsensusNewHeightEventType:
-		return node.GetBus().GetP2PModule().HandleEvent(message.Content)
+		if err := node.GetBus().GetP2PModule().HandleEvent(message.Content); err != nil {
```

Comment:
Look at `StateMachineTransitionEventType` and follow th same pattern leveraging `multierr.Combine`